-----------------
__main__
None
-----------------
C:\Users\wsutt\Desktop\files\chess\basic_engine\docs\py3_imports
C:\Users\wsutt\AppData\Local\Programs\Python\Python37\python37.zip
C:\Users\wsutt\AppData\Local\Programs\Python\Python37\DLLs
C:\Users\wsutt\AppData\Local\Programs\Python\Python37\lib
C:\Users\wsutt\AppData\Local\Programs\Python\Python37
C:\Users\wsutt\AppData\Roaming\Python\Python37\site-packages
C:\Users\wsutt\AppData\Local\Programs\Python\Python37\lib\site-packages
-----------------
subA.modB
subA
-----------------
C:\Users\wsutt\Desktop\files\chess\basic_engine\docs\py3_imports
C:\Users\wsutt\AppData\Local\Programs\Python\Python37\python37.zip
C:\Users\wsutt\AppData\Local\Programs\Python\Python37\DLLs
C:\Users\wsutt\AppData\Local\Programs\Python\Python37\lib
C:\Users\wsutt\AppData\Local\Programs\Python\Python37
C:\Users\wsutt\AppData\Roaming\Python\Python37\site-packages
C:\Users\wsutt\AppData\Local\Programs\Python\Python37\lib\site-packages
hello
the import in modA (of modB) failed, as expected
No module named 'modB'
-----------------
subA.modA
subA
-----------------
C:\Users\wsutt\Desktop\files\chess\basic_engine\docs\py3_imports
C:\Users\wsutt\AppData\Local\Programs\Python\Python37\python37.zip
C:\Users\wsutt\AppData\Local\Programs\Python\Python37\DLLs
C:\Users\wsutt\AppData\Local\Programs\Python\Python37\lib
C:\Users\wsutt\AppData\Local\Programs\Python\Python37
C:\Users\wsutt\AppData\Roaming\Python\Python37\site-packages
C:\Users\wsutt\AppData\Local\Programs\Python\Python37\lib\site-packages
hello
-----------------
the import failed as expected!
No module named 'subA.modA.foo2'; 'subA.modA' is not a package
